.github/workflows/ci.yml

```diff
@@ -121,21 +121,28 @@ jobs:
           repository: eclipse-velocitas/vehicle-app-${{ matrix.language }}-template
           path: test/${{ matrix.language }}/repo
 
+      - name: Replace project config
+        run: |
+          cp -f test/${{ matrix.language }}/.velocitas.json test/${{ matrix.language }}/repo
+
       - name: Add safe directory
         run: |
           git config --global --add safe.directory $(pwd)/test/${{ matrix.language }}/repo
 
-      - name: Update Velocitas JSON with correct version
-        uses: ./.github/actions/update-velocitas-json
-        with:
-          project_path: test/${{ matrix.language }}/repo
-          template_path: test/${{ matrix.language }}/.velocitas.json
+      - name: Setup local package version
+        shell: bash
+        run: |
+          export PACKAGE_PATH=~/.velocitas/packages/thispackage/main
```

Comment:
can you give "thispackage" a better name?

Reply:
Any suggestions? Its basically about "this package" being under test. Alternatively we can call it "put" or "package_under_test"

Reply:
"package_under_test" is fine